frontend/packages/core/package.json

```diff
@@ -35,6 +35,7 @@
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.58",
     "@material-ui/styles": "^4.11.4",
+    "@material-ui/system": "^4.12.1",
```

Comment:
i'd use a consistent version spec for all the mui libs, though i guess it wouldn't technically matter. if you need an upgrade we can do it separately.